resources/views/themes/default1/front/checkout.blade.php

```diff
@@ -165,49 +165,51 @@
                 <h4 class="heading-primary">Payment</h4>
              
                        
-                {!! Form::open(['url'=>'checkout','method'=>'post']) !!}
+                {!! Form::open(['url'=>'checkout','method'=>'post','id' => 'checkoutsubmitform' ]) !!}
 
                 @if(Cart::getTotal()>0)
                   
                  <?php 
                 $gateways = \App\Http\Controllers\Common\SettingsController::checkPaymentGateway($item['attributes']['currency']['currency']);
                 $total = Cart::getSubTotal();
                 $rzpstatus = \App\Model\Common\StatusSetting::first()->value('rzp_status');
+
                   // 
                 ?>
                 @if(count($gateways)>0 ) 
-                  <div class="form-group">
+                <div class="row">
+                  
 
                     <div class="col-md-6">
                         @foreach($gateways as $gateway)
-                        {{ucfirst($gateway)}} {!! Form::radio('payment_gateway',strtolower($gateway)) !!}<br><br>
+                        <?php
+                        $processingFee = \DB::table(strtolower($gateway))->where('currencies',$item['attributes']['currency']['currency'])->first()->processing_fee;
```

Comment:
`->first()` queries all the columns from the DB. If you need only `processing_fee`, you can use `->value('processing_fee')`method. It will query only that particular column

Reply:
Fixed